server/api/serializers/role.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from ..models import WorkspaceRole, MemberRole
+
+class RoleSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = WorkspaceRole
+        fields = ['name','pay_rate'] # workspace not included in serializer because it would need to be called 'workspace' rather than 'workspace_id', which would be inconsitent with other requests
```

Comment:
Can this be changed now that #20 has been merged?

Reply:
this cant be changed due to #20 being merged, since we use '_id' for request param names but not in the db.

Reply:
Ah I understand now. Makes sense. I'm sure there is some way around it but as long as it works correctly and efficiently as-is, it is fine.